posthog/api/cohort.py

```diff
@@ -283,13 +286,45 @@ def persons(self, request: Request, **kwargs) -> Response:
         elif not filter.limit:
             filter = filter.shallow_clone({LIMIT: 100})
 
-        query, params = PersonQuery(filter, team.pk, cohort=cohort).get_query(paginate=True)
+        if posthoganalytics.feature_enabled(
+            "load-person-fields-from-clickhouse",
+            None if request.user.is_anonymous else request.user.distinct_id,
+            person_properties={"email": request.user.email},
+        ):
+            person_query = PersonQuery(
+                filter,
+                team.pk,
```

Comment:
this seems to be missing the cohort filter, are you sure this is getting people for the cohort?

Reply:
Oops, fixed and updated test.